[2024/12/28 15:53:04.341] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.342] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.343] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.344] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.345] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.346] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.347] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.349] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.348] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.350] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.351] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.350] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.352] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.353] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.353] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.354] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.356] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.356] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.357] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.357] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.358] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.355] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.359] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.359] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.360] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.361] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.362] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.361] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:53:04.363] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.437] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.438] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.439] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.440] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.441] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.442] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.447] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.449] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.452] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.451] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.453] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.455] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.454] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.456] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.456] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.459] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.457] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.458] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.459] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.461] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.460] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.461] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.462] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.462] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.463] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.464] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.465] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.465] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:55:42.466] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.003] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.005] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.006] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.007] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.008] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.009] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.012] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.010] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.013] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.014] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.014] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.016] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.015] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.017] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.018] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.019] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.020] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.021] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.022] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.022] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.023] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.030] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.031] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.030] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.033] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.032] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.033] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:58:30.034] ERROR | Field 'content' doesn't have a default value
[2024/12/28 15:59:23.197] ERROR | 在同步Id为 315766035 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.199] ERROR | 在同步Id为 315764819 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.200] ERROR | 在同步Id为 315754001 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.201] ERROR | 在同步Id为 315764780 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.203] ERROR | 在同步Id为 315770747 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.202] ERROR | 在同步Id为 315770909 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.209] ERROR | 在同步Id为 315767626 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.212] ERROR | 在同步Id为 315770366 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.211] ERROR | 在同步Id为 315759344 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.213] ERROR | 在同步Id为 315757738 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.214] ERROR | 在同步Id为 315769558 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.215] ERROR | 在同步Id为 315758525 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.216] ERROR | 在同步Id为 312573657 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.217] ERROR | 在同步Id为 313700221 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.218] ERROR | 在同步Id为 315761763 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.219] ERROR | 在同步Id为 315763719 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.220] ERROR | 在同步Id为 315766338 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.220] ERROR | 在同步Id为 315765556 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.221] ERROR | 在同步Id为 315765618 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.222] ERROR | 在同步Id为 315758701 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.222] ERROR | 在同步Id为 315755353 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.223] ERROR | 在同步Id为 315756644 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.223] ERROR | 在同步Id为 315702460 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.224] ERROR | 在同步Id为 315757678 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.225] ERROR | 在同步Id为 315767042 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.225] ERROR | 在同步Id为 315761000 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.226] ERROR | 在同步Id为 315741377 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 15:59:23.227] ERROR | 在同步Id为 315766215 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value
[2024/12/28 16:00:22.591] ERROR | 在同步Id为 315766035 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.594] ERROR | 在同步Id为 315764819 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.596] ERROR | 在同步Id为 315754001 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.600] ERROR | 在同步Id为 315764780 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.602] ERROR | 在同步Id为 315770909 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.604] ERROR | 在同步Id为 315770747 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.606] ERROR | 在同步Id为 315767626 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.607] ERROR | 在同步Id为 315759344 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.609] ERROR | 在同步Id为 315770366 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.610] ERROR | 在同步Id为 315757738 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.612] ERROR | 在同步Id为 315769558 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.613] ERROR | 在同步Id为 315758525 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.616] ERROR | 在同步Id为 312573657 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.617] ERROR | 在同步Id为 313700221 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.619] ERROR | 在同步Id为 315761763 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.620] ERROR | 在同步Id为 315763719 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.622] ERROR | 在同步Id为 315766338 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.623] ERROR | 在同步Id为 315765556 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.624] ERROR | 在同步Id为 315765618 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.626] ERROR | 在同步Id为 315758701 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.627] ERROR | 在同步Id为 315755353 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.628] ERROR | 在同步Id为 315756644 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.630] ERROR | 在同步Id为 315702460 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.634] ERROR | 在同步Id为 315761000 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.633] ERROR | 在同步Id为 315757678 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.636] ERROR | 在同步Id为 315767042 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.638] ERROR | 在同步Id为 315766215 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:00:22.637] ERROR | 在同步Id为 315741377 的帖子数据到数据库中时发生了错误;错误信息为：Field 'content' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'content' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.687] ERROR | 在同步Id为 315767626 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.685] ERROR | 在同步Id为 315770909 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.689] ERROR | 在同步Id为 315766035 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.694] ERROR | 在同步Id为 315764780 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.690] ERROR | 在同步Id为 315764819 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.692] ERROR | 在同步Id为 315754001 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.695] ERROR | 在同步Id为 315770747 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.697] ERROR | 在同步Id为 315759344 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.698] ERROR | 在同步Id为 315770366 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.701] ERROR | 在同步Id为 315757738 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.702] ERROR | 在同步Id为 315769558 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.705] ERROR | 在同步Id为 315758525 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.707] ERROR | 在同步Id为 312573657 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.708] ERROR | 在同步Id为 313700221 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.709] ERROR | 在同步Id为 315761763 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.711] ERROR | 在同步Id为 315770138 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.712] ERROR | 在同步Id为 315763719 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.713] ERROR | 在同步Id为 315766338 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.714] ERROR | 在同步Id为 315765556 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.716] ERROR | 在同步Id为 315765618 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.717] ERROR | 在同步Id为 315758701 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.718] ERROR | 在同步Id为 315755353 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.719] ERROR | 在同步Id为 315756644 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.723] ERROR | 在同步Id为 315761000 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.721] ERROR | 在同步Id为 315702460 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.726] ERROR | 在同步Id为 315767042 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.722] ERROR | 在同步Id为 315757678 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.727] ERROR | 在同步Id为 315741377 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:04:45.728] ERROR | 在同步Id为 315766215 的帖子数据到数据库中时发生了错误;错误信息为：Field 'title' doesn't have a default value 
 stack:TypeORMWrappedError: QueryFailedError: Field 'title' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.019] ERROR | 在同步Id为 315767626 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.022] ERROR | 在同步Id为 315764819 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.021] ERROR | 在同步Id为 315766035 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.017] ERROR | 在同步Id为 315770909 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.028] ERROR | 在同步Id为 315770747 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.024] ERROR | 在同步Id为 315754001 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.026] ERROR | 在同步Id为 315764780 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.035] ERROR | 在同步Id为 315757738 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.032] ERROR | 在同步Id为 315770366 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.036] ERROR | 在同步Id为 315769558 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.029] ERROR | 在同步Id为 315759344 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.042] ERROR | 在同步Id为 313700221 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.040] ERROR | 在同步Id为 312573657 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.037] ERROR | 在同步Id为 315758525 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.044] ERROR | 在同步Id为 315761763 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.046] ERROR | 在同步Id为 315763719 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.045] ERROR | 在同步Id为 315770138 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.050] ERROR | 在同步Id为 315765556 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.051] ERROR | 在同步Id为 315765618 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.048] ERROR | 在同步Id为 315766338 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.054] ERROR | 在同步Id为 315755353 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.058] ERROR | 在同步Id为 315757678 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.053] ERROR | 在同步Id为 315758701 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.056] ERROR | 在同步Id为 315756644 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.057] ERROR | 在同步Id为 315702460 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.060] ERROR | 在同步Id为 315761000 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.061] ERROR | 在同步Id为 315767042 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.063] ERROR | 在同步Id为 315741377 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
[2024/12/28 16:07:22.064] ERROR | 在同步Id为 315766215 的帖子数据到数据库中时发生了错误;错误信息为：Field 'author_uid' doesn't have a default value 
 stack:lib.combinators.database.topic.upsertTopicContentSnapshot::TypeORMWrappedError: QueryFailedError: Field 'author_uid' doesn't have a default value
    at Query.onResult (G:\Github\DoubanTimeMachine\node_modules\typeorm\driver\src\driver\mysql\MysqlQueryRunner.ts:246:33)
    at Query.execute (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\commands\command.js:36:14)
    at PoolConnection.handlePacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (G:\Github\DoubanTimeMachine\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
